vtex-apps/store-icons
# Serve iconpack through styles builder

#### What is the purpose of this pull request?
- IconPack is served by `store-icons` instead of `store`
- Changed the name in manifest to `store-icons` instead of `dreamstore-icons`

#### How should this be manually tested?
Linked @ [workspace](https://iconsrefactor--storecomponents.myvtex.com). See that icons load normaly.

#### Obs
This should be merged alongside this [PR](https://github.com/vtex-apps/store/pull/186).
 
#### Types of changes
